core/ping/ping_common.go

```diff
@@ -50,10 +51,11 @@ func addResource(ping *revents.Event, pong *model.PingResponse, dockerClient *cl
 		logrus.Warnf("Failed to get Host Labels err msg: %v", err.Error())
 	}
 	rancherImage := os.Getenv(agentImage)
-	labels[constants.RancherAgentImage] = rancherImage
+	createdLabels := config.Labels()
```

Comment:
use the word "createLabels" not "createdLabels"
thats what theyre called incattle